dateutil/test/test_isoparser.py

```diff
@@ -286,15 +286,15 @@ def test_iso_with_sep_raises(sep_act, valid_sep, exception):
         parser.isoparse(isostr)
 
 
-@pytest.mark.xfail()
+@pytest.mark.xfail()                            #pragma: no cover
 @pytest.mark.parametrize('isostr,exception', [
     ('20120425T01:2000', ValueError),           # Inconsistent time separators
 ])
 def test_iso_raises_failing(isostr, exception):
```

Comment:
```suggestion
def test_iso_raises_failing(isostr, exception):    # pragma: nocover
```

Only this line needs `# pragma: nocover`. As I mention below, you can mark an entire function as not covered by a single pragma on the `def` line.